dist/s_evergreen_validate

```diff
@@ -3,25 +3,19 @@
 . `dirname -- ${BASH_SOURCE[0]}`/common_functions.sh
 setup_trap
 cd_top
+check_fast_mode_flag
 
 fast=false
 exit1=0
 exit2=0
-case "$1" in
-    -F) # Run fast.
-        check_fast_mode
-        fast=true
-        shift;;
-    *)
-        if test $# -gt 1 ; then
-            echo "Usage: $0 [-F]"
-            echo "-F only run validation commands if evergreen yml files have been modified"
-        fi
-        ;;
-esac
+if [[ -n "$1" && "$1" != "-F" ]]; then
+    echo "Usage: $0 [-F]"
+    echo "-F only run validation commands if evergreen yml files have been modified"
+    exit 1
+fi
 
 if [ $(command -v evergreen) ] && [ -f ~/.evergreen.yml ]; then
-    if $fast ; then
+    if is_fast_mode ; then
```

Comment:
Per the comment in `common_function.sh` I'm not sure the bash version of `is_fast_mode` is returning true/false. Please check this line works as expected